runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -368,15 +380,12 @@ public void onUserJoin(final UserJoin event)
 
 		if (localMember != null && partyData.getMemberId().equals(localMember.getMemberId()))
 		{
-			final String helpMessage = new ChatMessageBuilder()
-				.append(ChatColorType.HIGHLIGHT)
-				.append("To leave party hold SHIFT and right click party stats overlay.")
-				.build();
-
-			chatMessageManager.queue(QueuedMessage.builder()
-				.type(ChatMessageType.GAME)
-				.runeLiteFormattedMessage(helpMessage)
-				.build());
+			if (client.getGameState() != GameState.LOGGED_IN)
```

Comment:
just set the alrtFlag to true here, and process it in gameTicka nd set to false.